lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -486,25 +491,29 @@ bool UsdMaya_ReadJob::_DoImport(UsdPrimRange& rootRange, const UsdPrim& usdRootP
     }
 
     if (buildInstances) {
-        MDGModifier              deleteMasterMod;
+        MDGModifier              deletePrototypeMod;
         UsdMayaPrimReaderContext readCtx(&mNewNodeRegistry);
         readCtx.SetTimeSampleMultiplier(mTimeSampleMultiplier);
 
-        for (const auto& master : usdRootPrim.GetStage()->GetMasters()) {
-            const SdfPath masterPath = master.GetPath();
-            MObject       masterObject = readCtx.GetMayaNode(masterPath, false);
-            if (masterObject != MObject::kNullObj) {
+#if PXR_VERSION < 2011
+        for (const auto& prototype : usdRootPrim.GetStage()->GetMasters()) {
+#else
+        for (const auto& prototype : usdRootPrim.GetStage()->GetPrototypes()) {
+#endif
+            const SdfPath prototypePath = prototype.GetPath();
+            MObject       prototypeObject = readCtx.GetMayaNode(prototypePath, false);
+            if (prototypeObject != MObject::kNullObj) {
                 MStatus    status;
-                MFnDagNode masterNode(masterObject, &status);
+                MFnDagNode prototypeNode(prototypeObject, &status);
                 if (status) {
-                    while (masterNode.childCount()) {
-                        masterNode.removeChildAt(masterNode.childCount() - 1);
+                    while (prototypeNode.childCount()) {
+                        prototypeNode.removeChildAt(prototypeNode.childCount() - 1);
                     }
                 }
-                deleteMasterMod.deleteNode(masterObject);
+                deletePrototypeMod.deleteNode(prototypeObject);
             }
         }
-        deleteMasterMod.doIt();
+        deletePrototypeMod.doIt();
```

Comment:
I found both masterObject and prototypeObject confusing. Are they not just "instance objects"? I think having a more generic name would be better.

Reply:
I've made the other changes as it breaks functionality, but I've left this for now since it's a style preference. If you feel strongly about changing the var names, I can raise it with the USD team. Please let me know.